src/doom/st_stuff.c

```diff
@@ -407,25 +407,16 @@ void ST_refreshBackground(boolean force)
 	// so it appears to the left and right of the status bar in widescreen mode
 	if ((SCREENWIDTH >> crispy->hires) != ST_WIDTH)
 	{
-		int x, y;
+		int x;
```

Comment:
Could you move the `x` into its own scope inside the `if (scaledviewwidth == SCREENWIDTH)` check?